scripts/intersections.py

```diff
@@ -0,0 +1,73 @@
+import configparser
+import os
+import warnings
+import geopandas as gpd
+import pandas as pd
+pd.options.mode.chained_assignment = None
+warnings.filterwarnings('ignore')
+
+CONFIG = configparser.ConfigParser()
+CONFIG.read(os.path.join(os.path.dirname(__file__), 'script_config.ini'))
+BASE_PATH = CONFIG['file_locations']['base_path']
+
+DATA_RAW = os.path.join(BASE_PATH, 'raw')
+DATA_PROCESSED = os.path.join(BASE_PATH, 'processed')
+DATA_RESULTS = os.path.join(BASE_PATH, 'results')
+
+path = os.path.join(DATA_RAW, 'countries.csv')
+
+# Population and floods layer
+population_folder = os.path.join(BASE_PATH, 'processed', 'KEN', 'population', 'shapefiles')
```

Comment:
So you need to make this so that it's generic, and not 'KEN'. Thus, you need a generic function which takes an iso3, and potentially also the regional gid_id depending on how you do it.